specification/communication/Communication.Messages/models.tsp

```diff
@@ -42,6 +42,58 @@ union CommunicationMessageKind {
 
   @doc("Template message type.")
   template: "template",
+
+  @added(Versions.c2024_11_15_Preview)
+  @doc("Sticker message type.")
+  sticker: "sticker",
+
+  @added(Versions.c2024_11_15_Preview)
+  @doc("Reaction message type.")
+  reaction: "reaction",
+
+  @added(Versions.c2024_11_15_Preview)
+  @doc("Interactive Actionable message type.")
+  interactive: "interactive",
+}
+
+@doc("The Kind of interactive message channel action binding like WhatsAppListAction.")
+union MessageActionBindingKind {
```

Comment:
Should we use `@added` to the 2 new union?

Reply:
Added